src/Symfony/Component/Security/Http/EventListener/IsCsrfTokenValidAttributeListener.php

```diff
@@ -49,4 +56,18 @@ public static function getSubscribedEvents(): array
     {
         return [KernelEvents::CONTROLLER_ARGUMENTS => ['onKernelControllerArguments', 25]];
     }
+
+    private function getIsCsrfTokenValidId(string|Expression $id, Request $request, array $arguments): string
+    {
+        if ($id instanceof Expression) {
+            $this->expressionLanguage ??= new ExpressionLanguage();
```

Comment:
I'd throw a LogicExpression there

Reply:
I did exactly like in [IsGrantedAttributeListener](https://github.com/symfony/symfony/blob/7.1/src/Symfony/Component/Security/Http/EventListener/IsGrantedAttributeListener.php#L86)

Reply:
(just to be very nitpicky) 

Did you _exactly_ ? :) 

https://github.com/symfony/symfony/blob/3f2ed0fc4c41744991113185a619baf5d2f7590f/src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php#L122
```
    $container->removeDefinition('security.is_granted_attribute_expression_language');
```

Reply:
very valid nitpick! :)